velox/exec/GroupingSet.cpp

```diff
@@ -725,6 +725,7 @@ bool GroupingSet::getOutput(
   }
 
   if (hasSpilled()) {
+    spill();
```

Comment:
If we want to make spilling and flushing work together, `getOutput` may have to flush in-memory and spilled data together. In the case we should spill in-memory data out then call `getOutputWithSpill`.